markdown/report.Rmd

```diff
@@ -204,7 +237,7 @@ paste("All data are revised to ensure the most complete data are presented. Data
 "Policy concepts; [http://www.gov.scot/Topics/Health/Policy/Quality-Strategy/Quality-Measurement-Framework](http://www.gov.scot/Topics/Health/Policy/Quality-Strategy/Quality-Measurement-Framework), Methodology; [Appendix 2](#appendix-2---background-information) – Background Information, [Appendix 3](#appendix-3---methodology) - Methodology",
 "Making information publicly available; Allowing assessment of implementation/impact of policies or targets ",
 "SMR01 and SMR04 standard validation processes; Wide stakeholder consultation on technical methodology",
-paste("SMR01 hospital completeness for the financial year", latest$fy, "is estimated to be 87% complete and this could have an impact on the time spent in hospital in the last six months of life. NHS Forth Valley and NHS Greater Glasgow & Clyde are particularly impacted (completeness for full financial year", latest$fy, "is estimated as 73% and 76% respectively). See [Appendix 2](#appendix-2---background-information) for more information and completeness figures for each quarter. Further updates  will be made available at the following link:  [http://www.isdscotland.org/Products-and-Services/Data-Support-and-Monitoring/SMR-Completeness/](http://www.isdscotland.org/Products-and-Services/Data-Support-and-Monitoring/SMR-Completeness/)"),
+paste("SMR01 hospital completeness for the financial year", latest$fy, "is estimated to be", comp_scot, "complete and this could have an impact on the time spent in hospital in the last six months of life.", comp_metadata, "See [Appendix 2](#appendix-2---background-information) for more information and completeness figures for each quarter. Further updates  will be made available at the following link:  [http://www.isdscotland.org/Products-and-Services/Data-Support-and-Monitoring/SMR-Completeness/](http://www.isdscotland.org/Products-and-Services/Data-Support-and-Monitoring/SMR-Completeness/)"),
 paste("Due to a change in methodology, results presented in this publication will not be comparable to figures published prior to August 2016, Trend information from 2010/11 to", latest$fy, "is available in this publication."),
 "It is the policy of ISD Scotland to make its web sites and products accessible according to published guidelines.",
 "Publication is made available via the ISD website.",
```

Comment:
Can't leave a comment for next published date as it has been approved. I noticed you use "next_pub_date" for Next published. I think it's correct, although by having a look at the May and October 2019 publication they are both the date of current publication (e.g. for May 2019 publication, the Next published is 28/05/2019) - just want to double check which date you want to use?

Reply:
How weird! - I've not noticed this before. I'll double check with Ag, but I definitely think this is correct as is and should be date of next publication.